csm/core/controllers/s3/access_keys.py

```diff
@@ -69,6 +69,11 @@ async def get(self):
             resp = await self._service.list_access_keys(self._s3_session, **request_url_data)
             filtered_keys = [k for k in resp['access_keys'] if k['access_key_id'] not in tmp_keys]
             resp['access_keys'] = filtered_keys
+            # change the user_name, root to s3username in case of s3user request for accesskeys
+            if request_url_data.get('user_name') is None:
```

Comment:
Move all the static strings to const.py

Reply:
I have made the suggested changes @pawankumarsrivastava